dap-mode.el

```diff
@@ -1335,6 +1335,29 @@ RESULT to use for the callback."
              error))
     (error "No thread is currently active %s" (dap--debug-session-name (dap--cur-session)))))
 
+(defun dap-up-stack-frame (frames)
+  "Switch stackframe up FRAMES frames on the current thread.
+A negative value will move down frames."
+  (interactive "p")
+  (-if-let* ((session (dap--cur-session))
+             (thread-id (dap--debug-session-thread-id session))
+             (stack-frames (gethash thread-id
+                                    (dap--debug-session-thread-stack-frames session)))
+             (cur-frame (dap--debug-session-active-frame session))
+             (pos (position cur-frame stack-frames))
```

Comment:
CI breaks, please use cl-position.

Reply:
Oops. I guess it's time for me to remove the `(require 'cl)` from my .emacs.